### ISSUE TITLE
CETO Ocean shows sun reflection across the water, even if sun is occluded behind a big terrain

### ISSUE THREAD

**Issue by muzboz** [Scrawk/Ceto#12]
Is there a way to have the light reflection of the sun across the water be blocked and not rendered, if the sun light source itself is behind an object, such as a big terrain?

**Comment by MrUnbelievable92** [comment]
Unfortunately, when you get close to high waves, you can see "reflections" through the water as if it were see-through instead. Kinda trippy to be honest:
![alt text](https://i.imgur.com/psLTMo0.jpg)

**Comment by muzboz** [comment]
Yeah, it's a shame about some artifacts and issues, because overall I just love CETO!  :)

Your images there show some of the undesirable elements.

Oh well, I think I can make do with how it is!  :D
